Sources/TuistKit/Commands/Cloud/CloudProjectListCommand.swift

```diff
@@ -0,0 +1,26 @@
+import ArgumentParser
+import Foundation
+import TSCBasic
+import TuistSupport
+
+struct CloudProjectListCommand: AsyncParsableCommand {
+    static var configuration: CommandConfiguration {
+        CommandConfiguration(
+            commandName: "list",
+            _superCommandName: "project",
+            abstract: "List projects you have access to."
```

Comment:
Does it include all the projects of all the organizations a user belongs to?

Reply:
Yes, that's correct. I think that's ok for now, eventually, we can add filters. But for the time being, I don't expect developers to have too many projects.